desktop/desktop.scss

```diff
@@ -77,6 +77,11 @@
     }
 }
 
+//remove stray border from bottom of last post
+.topic-status-info {
+    border-top: 0;
```

Comment:
I'm confused - I looked at the lines above and they're 4 spaced (which is why I assumed the indentation standard was 4 spaces), but at the top they're 2 spaced.